lib/stm32/f4/rcc.c

```diff
@@ -307,6 +357,22 @@ const struct rcc_clock_scale rcc_hse_25mhz_3v3[RCC_CLOCK_3V3_END] = {
 		.apb1_frequency = 42000000,
 		.apb2_frequency = 84000000,
 	},
+    { /* 96MHz */
+		.pllm = 25,
+		.plln = 192,
```

Comment:
hang on, is this just more clock configs for 96mhz, mixed in with a commit that is for enabling convenience functions for HSI?  that's confusing.  Also, what's the particular motivation for 96mhz configs anyway?  Is that the max speed of any parts?  Custom configs like this can just be created in user code.